vhdl_parser/src/ast.rs

```diff
@@ -948,6 +970,7 @@ pub enum ConfigurationDeclarativeItem {
 pub struct ComponentConfiguration {
     pub spec: ComponentSpecification,
     pub bind_ind: Option<BindingIndication>,
+    pub vunit_bind_inds: Option<Vec<VUnitBindingIndication>>,
```

Comment:
Maybe Option<Vec> could just be a Vec. None would be an empty Vec.

Reply:
Ok, makes sense. I'll remove it.